### PR TITLE
fix: Remove ORPHANED status from module and update resource API version - `avm/res/network/front-door`

### PR DESCRIPTION
**Pull request by hundredacres**
## Description
Remove ORPHANED file from module.
Update resource API version.

## Pipeline Reference

| Pipeline |
| -------- |
|  [avm.res.network.front-door](https://github.com/hundredacres/bicep-registry-modules/actions/workflows/avm.res.network.front-door.yml)        |

## Type of Change

- [ ] Update to CI Environment or utilities (Non-module affecting changes)
- [ ] Azure Verified Module updates:
  - [ ] Bugfix containing backwards-compatible bug fixes, and I have NOT bumped the MAJOR or MINOR version in `version.json`:
    - [ ] Someone has opened a bug report issue, and I have included "Closes #{bug_report_issue_number}" in the PR description.
    - [ ] The bug was found by the module author, and no one has opened an issue to report it yet.
  - [ ] Feature update backwards compatible feature updates, and I have bumped the MINOR version in `version.json`.
  - [ ] Breaking changes and I have bumped the MAJOR version in `version.json`.
  - [ ] Update to documentation

## Checklist

- [X] I'm sure there are no other open Pull Requests for the same update/change
- [X] I have run `Set-AVMModule` locally to generate the supporting module files.
- [ ] My corresponding pipelines / checks run clean and green without any errors or warnings

### CODE REVIEW

File: .github/workflows/avm.res.fabric.capacity.yml [hunk 1]
```diff
@@ -0,0 +1,88 @@
+name: "avm.res.fabric.capacity"
```
Comment (krbar) [inline .github/workflows/avm.res.fabric.capacity.yml:1]: Hi, I guess this file doesn't belong here, please remove it from PR.

File: avm/res/network/front-door/main.bicep [hunk 2]
```diff
@@ -101,7 +101,7 @@ resource avmTelemetry 'Microsoft.Resources/deployments@2024-03-01' = if (enableT
   }
 }
 
-resource frontDoor 'Microsoft.Network/frontDoors@2020-05-01' = {
+resource frontDoor 'Microsoft.Network/frontDoors@2021-06-01' = {
```
Comment (krbar) [inline avm/res/network/front-door/main.bicep:104]: If this PR needs to include the API version update, I recommend indicating it in the PR title.

**Review by krbar — CHANGES_REQUESTED** [review]
Would you kindly include a pipeline badge as well?

File: .github/workflows/avm.res.fabric.capacity.yml (hunk 1 above)
Comment (hundredacres) [reply .github/workflows/avm.res.fabric.capacity.yml:1]: Fixed.

**Comment by hundredacres** [comment]
> Would you kindly include a pipeline badge as well?

Included pipeline badge. I believe this might fail, at least it did when I tried this earlier due to the module not being identified as un-ORPHANED.

`Error: [-] [network/front-door] Module should contain a [ ORPHANED.md ] file only if orphaned. 999ms (996ms|4ms)`

File: avm/res/network/front-door/main.bicep (hunk 2 above)
Comment (hundredacres) [reply avm/res/network/front-door/main.bicep:104]: Done

**Review by krbar — APPROVED** [review]

**Pull request merged**